services/metalv1/model_vlan_virtual_circuit.go

```diff
@@ -22,13 +22,14 @@ var _ MappedNullable = &VlanVirtualCircuit{}
 // VlanVirtualCircuit struct for VlanVirtualCircuit
 type VlanVirtualCircuit struct {
 	// True if the Virtual Circuit is being billed. Currently, only Virtual Circuits of Fabric VCs (Metal Billed) will be billed. Usage will start the first time the Virtual Circuit becomes active, and will not stop until it is deleted from Metal.
-	Bill        *bool   `json:"bill,omitempty"`
-	Description *string `json:"description,omitempty"`
-	Id          *string `json:"id,omitempty"`
-	Name        *string `json:"name,omitempty"`
-	NniVlan     *int32  `json:"nni_vlan,omitempty"`
-	Port        *Href   `json:"port,omitempty"`
-	Project     *Href   `json:"project,omitempty"`
+	Bill        *bool                              `json:"bill,omitempty"`
+	BillType    NullableVlanVirtualCircuitBillType `json:"bill_type,omitempty"`
```

Comment:
Interesting type introduced here: `NullableVlanVirtualCircuitBillType`

Reply:
Curious if Nullable is a necessary distinction or if the spec for the attribute is more verbose in definition than others and this is what that looks like.

Without digging into this, I would have expected:
```
BillType *VlanVirtualCircuitBillType
```

Reply:
https://github.com/equinix/equinix-sdk-go/pull/10/files#diff-5d2665305836606e9511c3789f7e4a11d1d60ef29693a406ec0d2b1629eefbc0R8